com.unity.netcode.gameobjects/Utilities/TestHelpers/NetworkManagerHelper.cs

```diff
@@ -96,6 +95,11 @@ public static bool StartNetworkManager(out NetworkManager networkManager, Networ
             return true;
         }
 
+        internal static void ThrowException(string message)
```

Comment:
Curious why this was made into a function? It doesn't save much in terms of typing and `throw new Exception()` is pretty standard. I thought at first maybe it was to be able to pass this to a callback, but I don't see that happening so I'm a little confused as to this design choice.

It's not doing anything wrong, per se, but reading the code, it strikes me as odd.